package.json

```diff
@@ -22,11 +22,11 @@
     "lint": "npm run lint:javascript && npm run lint:typescript",
     "lint:javascript": "eslint index.js esm.mjs \"lib/*.js\" \"tests/**/*.js\"",
     "lint:typescript": "eslint typings/*.ts tests/*.ts",
-    "test": "jest && npm run test-typings",
+    "test": "jest && npm run typecheck-ts",
     "test-esm": "node ./tests/esm-imports-test.mjs",
-    "test-typings": "tsd",
-    "typescript-checkJS": "tsc --allowJS --checkJS index.js lib/*.js --noEmit",
-    "test-all": "npm run test && npm run lint && npm run typescript-checkJS && npm run test-esm"
+    "typecheck-ts": "tsd && tsc -p tsconfig.ts.json",
+    "typecheck-js": "tsc -p tsconfig.js.json",
```

Comment:
How about `typecheck:ts` instead of `typecheck-ts`?
We already use `lint:typescript`.

Reply:
I have been following a pattern I saw and liked long ago, but it is arbitrary. I use the colon for sub-scripts which are the parts of a combo script. So `lint:typescript` is one part of `lint`.

```
"lint": "npm run lint:javascript && npm run lint:typescript",
```